app/react/docker/containers/CreateView/CreateView.tsx

```diff
@@ -189,11 +189,10 @@ function useOldContainer(initialName?: string) {
       name: [`^/${debouncedName}$`],
     },
   });
+
   useEffect(() => {
-    if (initialName && initialName !== name) {
-      setName(initialName);
-    }
-  }, [initialName, name]);
+    setName(initialName);
+  }, [initialName]);
```

Comment:
what if the user already typed a name before the old container is loaded?

```
  useEffect(() => {
     if (typeof name  !== "undefined") {
       setName(initialName);
     }
  }, [initialName, name]);
```

Reply:
the user should has no opportunity to type a name as the UI has not been rendered when the initial value is null.

Reply:
currently. but what if in the future we change this behavior? we'll need to remember to add this condition here. why not just add it?